File: backend/Odevler/Gun_7/Odev_1/hrms/src/main/java/javaReact_BootCamp/hrms/business/abstracts/UserService.java
package javaReact_BootCamp.hrms.business.abstracts;

import java.util.List;

import javaReact_BootCamp.hrms.core.utilites.result.DataResult;
import javaReact_BootCamp.hrms.core.utilites.result.Result;
import javaReact_BootCamp.hrms.entities.concretes.User;

public interface UserService {
	DataResult<List<User>> getAll();
	DataResult<User> login(String email,String password);
	
	DataResult<User> checkEmail(String email);
	
}

File: frontend/hrms/hrms/src/services/userService.js
import axios from 'axios'

export default class UserService{
    login(props){
        return axios.get(`http://localhost:8080/api/users/login?email=${props.email}&password=${props.password}`)
    }

    checkEmail(email){
        return axios.get(`http://localhost:8080/api/users/checkemail?email=${email}`)
    }
}
